api_intro.py

```diff
@@ -1,24 +1,31 @@
 from datetime import date
+import json
 
 import requests
 
-import config
+# Get configuration data from config.json
+try:
+    with open("config.json", "r") as config_file:
+        config_data = json.load(config_file)
+        config_file.close()
+except FileNotFoundError:
```

Comment:
I think it is possible to raise other different exceptions, e.g. `JSONDecodeError` or `PermissionError`. Make sure your script handles all of them properly and you also need to catch the generic base exception for unexpected internal error as well. This is something you have to be careful about when doing the API development as well.

Reply:
These have been added for both try/except blocks.